situp-plugins/elasticsearch/src/main/resources/otel-v1-apm-span-index-template.json

```diff
@@ -38,9 +49,27 @@
       "endTime": {
         "type": "date_nanos"
       },
-      "resource.attributes.service.name": {
-        "ignore_above": 1024,
+      "status": {
+        "properties": {
+          "code": {
+            "type": "integer"
+          },
+          "message": {
+            "type": "keyword"
+          }
+        }
+      },
+      "serviceName": {
         "type": "keyword"
+      },
+      "durationInNanos": {
+        "type": "long"
+      },
+      "events": {
+        "type": "nested"
+      },
+      "links": {
+        "type": "nested"
```

Comment:
Just asking for my clarification, this definition will fail if index request is anyone of the following,
1. events: "some string"
2. events: { "atrributes": { "key":"value"}}

Reply:
Yeah. If the above 2 coexists, define `events` as `nested` will lead to error. I do not know if any field type will work for the above scenario.

Reply:
So the above 2 examples are both invalid, is there way to force and allow only,
`events: [ { time : ".....",  attributes : { key : "value" } } ]`

Reply:
Let me check path_match for nested